PeerioTechnologies/peerio-desktop
# style: input focusing in UserPicker user search

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/18192/desktop-click-area-too-small-in-new-search-field

#### Testing instructions
In "Add Members", clicking anywhere inside the user search border should focus the text input. This will work anywhere the UserPicker is used actually, but is most relevant for that screen.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
